nbqa/config_files/__init__.py

```diff
@@ -0,0 +1 @@
+"""Configuration files."""
```

Comment:
We could just rename the package to `config` and also put the modules for the config here.